### PR TITLE
Exclude 3 month recent registrations

### PR DESCRIPTION
**Pull request by timcheadle**
**Story card:** https://app.clubhouse.io/simpledotorg/story/764/update-my-facilities-to-exclude-patients-registered-in-last-3-months

## Because

We want to consistency exclude patients who registered in the last 3 months from control rates

## This addresses

Changes My Facilities from a 2 month to a 3 month recent registration buffer.

### CODE REVIEW

**Comment by rsanheim** [comment]
@dburka @claudiovallejo We should wait to ship this until we have the same change done for all the new dashboards. We also need some UI work to explain what is going on when we do that, because the graphs are gonna look a little wonky.

File: app/views/my_facilities/blood_pressure_control.html.erb [hunk 1]
```diff
@@ -95,7 +95,7 @@
           <th class="row-label sort-label" data-sort-method="number" data-sort-column-key="missed" colspan="2">No BP taken<br>in <%= visited_in_period %>
           </th>
           <th class="row-label sort-label" data-sort-method="number" data-sort-column-key="overall-percent" colspan="2">Controlled BP<br>in last 90 days</th>
-          <th class="row-label sort-label" data-sort-method="number" data-sort-column-key="overall-registered">Registered<br>&gt; 2 months ago</th>
```
Comment (rsanheim) [inline app/views/my_facilities/blood_pressure_control.html.erb:98]: This should use the same constant you modified above.

File: app/views/my_facilities/blood_pressure_control.html.erb [hunk 2]
```diff
@@ -95,7 +95,7 @@
...
+          <th class="row-label sort-label" data-sort-method="number" data-sort-column-key="overall-registered">Registered<br>&gt; 3 months ago</th>
```
Comment (prabhanshuguptagit) [inline app/views/my_facilities/blood_pressure_control.html.erb:98]: https://github.com/simpledotorg/simple-server/pull/1206/files#diff-a60da1211f63a657a368282d310b62f5R157


This footnote down below also needs change.

Comment (prabhanshuguptagit) [reply app/views/my_facilities/blood_pressure_control.html.erb:98]: Addressed in 5b0999e.

**Comment by kitallis** [comment]
@timcheadle @rsanheim Do we need to do anything here? Can this go in?

**Comment by harimohanraj89** [comment]
Now that Reports do the 3 month exclusion, this should be merged as well. Ideally we push all of this together in a single release.

**Pull request merged**